Components/Images/images.css

```diff
@@ -0,0 +1,29 @@
+@import url("/CSS/main.css");
+/* * {
+  outline: 1px solid red;
+} */
+.image-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 1rem;
+  margin: 2rem 0;
+  border: 2px solid var(--light-grey);
+  align-items: baseline;
+  border-radius: 0.3rem;
+  width: 100%;
+}
+.responsive-img {
+  width: 100%;
+  height: auto;
+}
+.r-image {
```

Comment:
Classname `r-image` isn't very descriptive. So maybe try using class names that are self-descriptive like `responsive-img`, `round-image` etc depending of what your class is doing